.gitignore

```diff
@@ -1,2 +1,4 @@
 *.retry
-cakephp-ex
\ No newline at end of file
+cakephp-ex
+config.yml
```

Comment:
Line 2 and 3 are not needed since config yml is dropped and cakephp-ex is deployed from remote template, not from local git repo